File: 1078. Occurrences After Bigram.py
class Solution:
    def findOcurrences(self, text: str, first: str, second: str) -> List[str]:
        l = []
        lst = text.split(" ")
        for i in range(len(lst)-2):
            if lst[i]==first and lst[i+1]==second:
                l.append(lst[i+2])
        return l

File: 1078. Occurrences After Bigram.cs
public class Solution {
    public string[] FindOcurrences(string text, string first, string second) {
        string[] s = text.Split(" ");
        List<string> x = new List<string>();
        for(int i=0;i<s.Length-2;i++){
            if(s[i]==first && s[i+1]==second){
                x.Add(s[i+2]);
            }
        }
        return x.ToArray();

    }
}
